.github/workflows/ci.yaml

```diff
@@ -74,9 +74,9 @@ jobs:
           brew install graphviz
           # Do not let tools interfere with the main module's go.mod.
           cd && go mod init tools
-          go get -u honnef.co/go/tools/cmd/... github.com/golangci/golangci-lint/cmd/golangci-lint@v1.50.1
-          go install honnef.co/go/tools/cmd/...
-          go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.50.1
+          # TODO: Update to a specific version when https://github.com/dominikh/go-tools/issues/1362 is fixed.
```

Comment:
Looks like this issue has been fixed, pending releases. If the release happened before this PR submission, we can remove this TODO and point to the latest version.

Reply:
Reading https://github.com/dominikh/go-tools/issues/1360 I am not sure there is a release for this yet, I asked. Submitting to get the CI fixed, I will update the version separately.